packages/dashboard/src/components/MultiCount/MultiCount.types.ts

```diff
@@ -35,6 +35,17 @@ export interface IMultiCountProps {
    *Url to be redirected to upon clicking on the component
    */
   href?: string;
+
+  /**
+   * Defines the id for the component
+   */
+  id?: string;
+
+  /**
+   * Defines the function that is executed on clicking this action
+   */
+
+  onClicked?: VoidFunction;
```

Comment:
I believe simply adding `onClicked` won't make it work, we also need to call it from the component.

Reply:
What does this onClick do?